themes/default/layouts/page/developer-advocates.html

```diff
@@ -0,0 +1,54 @@
+{{ define "hero" }}
+    {{ partial "hero" (dict "title" "Developer Advocates") }}
+{{ end }}
+
+{{ define "main" }}
+    <section class="max-w-4xl container mx-auto py-8 px-4 text-center text-lg">
+        <p>
+            The Pulumi Developer Advocates do things to help you be awesome. Here's a list of the team
+            and their social media contact info! We would love to help you be super cool and rad!

```

Comment:
Suggestion to match the current messaging a bit more, and narrowing the focus a bit so people aren't encouraged to reach out for not-advocate reasons.

```suggestion
            The Pulumi Developer Advocate team wants to help you build, manage, and deploy
            all the things and learn something new along the way. Here's a list of the team
            and their social media. Need help? We would love to help you accomplish your goals!
```